recipes/tigervnc/meta.yaml

```
@@ -0,0 +1,114 @@
+{% set name = "tigervnc" %}
+{% set version = "1.12.0" %}
+{% set xorg_x11_server_version = "1.20.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/TigerVNC/tigervnc/archive/v{{ version }}.tar.gz
+    sha256: 9ff3f3948f2a4e8cc06ee598ee4b1096beb62094c13e0b1462bff78587bed789
+    patches:
+      # Disables vncsession (requires PAM and runs as root)
+      # Disable PAM password check (always return failure)
+      - ignore-pam.patch
+  # https://github.com/TigerVNC/tigervnc/blob/v1.12.0/BUILDING.txt#L100-L146
+  - url: https://github.com/freedesktop/xorg-xserver/archive/xorg-server-{{ xorg_x11_server_version }}.tar.gz
+    sha256: 68105526da70d04f7111339044cdd5b2a762717178cd345aba6cb2d1b399c71b
+    folder: build/unix/xserver/
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - make
+    # For building Xvnc
+    - autoconf
+    - automake
+    - patch
+    - libtool
+    - pkgconfig
+    - xorg-font-util
+    - xorg-util-macros
+    # For building Xvnc (needs to be here and in host)
+    - libxkbfile
+    - xorg-bigreqsproto
+    - xorg-compositeproto
+    - xorg-damageproto
+    - xorg-fontsproto
+    - xorg-glproto
+    - xorg-presentproto
+    - xorg-randrproto
+    - xorg-recordproto
+    - xorg-resourceproto
+    - xorg-scrnsaverproto
+    - xorg-videoproto
+    - xorg-xcmiscproto
```

According to https://github.com/conda-forge/staged-recipes/pull/14990#issue-897301389, all the X.org "proto" packages has been combined to `xorgproto`. Not sure if this works, but could test it out.

```suggestion
    - xorg-xorgproto
```